widgetastic_manageiq/__init__.py

```diff
@@ -4509,6 +4509,10 @@ class MigrationPlanRequestDetailsList(Widget):
         './/div[@class="modal-footer"]' '/button[text()="No"]'
     )
     ITEM_MODAL_CANCEL_MIGRATION_VM_LIST_LOCATOR = './/div[@class="modal-body"]/ul'
+    ITEM_NOTIFICATION_TOAST_DISMISS_BUTTON_LOCATOR = (
```

Comment:
Optional : 
Locator name need not have word locator in it .

ITEM_NOTIFICATION_TOAST_DISMISS_BUTTON = (
 './/div[contains(@class,"alert-dismissable")]/button/span[contains(@class,"pficon-close")]')

Reply:
will re-factor in a separate PR.